Game/MyProject/Game/SceneManagement/SceneNode.hpp

```diff
@@ -39,6 +39,8 @@ class SceneNode : std::enable_shared_from_this<SceneNode> {
     const IvMatrix44& getAbsoluteTransform() const;
     IvVector3 getAbsolutePosition() const;
     const BoundingBox& getBoundingBox() const;
+    const IvVector3& getLocalScale() const;
+    void setParent(const std::shared_ptr<SceneNode>& parent);
```

Comment:
dangerous function because the node owner changes without children changed in the old parent. remove it pls.